docs/data-sources/tozny_realm_application_saml_description.md

```diff
@@ -138,8 +138,9 @@ data "tozny_realm_application_saml_description" "saml_xml_description" {
 
 ### Top-Level Arguments
 
-* `client_credentials_filepath` - (Optional) The filepath to Tozny client credentials for the Terraform provider to use when fetching this application's SAML description.
-* `description` - (Computed) OIDC Client secret for the application.
+* `client_credentials_filepath` - (Optional) The filepath to Tozny client credentials for the Terraform provider to use when fetching this application's SAML description. Omit if using `client_credentials_config`.
+* `client_credentials_config` - (Optional) A JSON string containing Tozny client credentials for the provider to use
+* `description` - (Computed) OIDC Client secret for the application.s application's SAML description. Omit if using `client_credentials_filepath`.
```

Comment:
my bug, but description comment looks wrong as this is neither OIDC or a secret. Regarding the omit bit, you can use https://www.terraform.io/docs/extend/schemas/schema-methods.html `ConflictsWith` to enforce this "magic"